core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -120,26 +119,11 @@ export class DotEditLayoutComponent implements OnInit, OnDestroy {
      * @param {DotLayout} value
      * @memberof DotEditLayoutComponent
      */
-    nextUpdateTemplate(value: DotTemplateDesignerPayload) {
+    nextUpdateTemplate(value: DotTemplateDesigner) {
         this.canRouteBeDesativated(false);
         this.updateTemplate.next(value);
     }
 
-    /**
-     * Handle layout change event.
-     * Take the layout out, update the template and save it.
-     *
-     * @param {DotLayout} layout
-     * @memberof DotEditLayoutComponent
-     */
-    onLayoutChange(layout: DotLayout) {
-        this.nextUpdateTemplate({
-            layout,
-            themeId: this.pageState.template.theme,
-            title: this.pageState.page.title
-        });
-    }
-
```

Comment:
That code was created in this PR: https://github.com/dotCMS/core/pull/25210  [[In this Line](https://github.com/dotCMS/core/commit/4fc3169b0077eb71868758e46353f0b150ad77b6#diff-7210dd13d57a153654b617dd8d9bd5da42258450ec5da582702235d2f97a3ea3R135-R140)] because in the `new template builder`, we were emitting only the `layout` but the backend needs an object with the `layout`, `themeId`, and `title`.